src/lib/jsonParseV2/CMakeLists.txt

```diff
@@ -32,13 +32,17 @@ SET (SOURCES
     parseSubscription.cpp
     parseBatchQuery.cpp
     parseEntityVector.cpp
+    parseEntitiesVector.cpp
```

Comment:
There is some "asymetric" in the additions to this file. I've seem 4 .cpp files added, but only 3 .h files. Maybe is ok, but I'm raising the comment just in case.

Reply:
Yeah ...
The thing is ... we don't use the header list for anything at all ... Not sure why we maintain it.
To me it seems use ful if you need to copy all header files of a library to somewhere, but that is not really the case, right?
I will fix this though, now that we have it, we should keep it up to date ...

Reply:
Fixed in 1a49367